packages/cli/src/commands/init/init.ts

```diff
@@ -490,7 +527,7 @@ async function createProject(
   // +------------------------------------------------------------------+----------+--------------+
   // | --template @react-native-community/template@0.75.1 --version 0.75| 0.75.1   | 0.75.x       |
   // +------------------------------------------------------------------+----------+--------------+
```

Comment:
Let's update this table with the latest logic also?

Reply:
If we keep the option of having `--template` and `--version` (with a warning if you use both), then this table + comments are now accurate.